shared/domains/chelonia/encryptedData.js

```diff
@@ -295,6 +295,7 @@ export const isRawEncryptedData = (data: any): boolean => {
 
 export const unwrapMaybeEncryptedData = (data: any): { encryptionKeyId: string | null, data: any } | void => {
   if (isEncryptedData(data)) {
+    if (process.env.BUILD !== 'web') return
```

Comment:
Where is this envvar set? (probably could use a comment mentioning that too)

Reply:
It's set in Gruntfile.js